package.json

```diff
@@ -39,7 +39,7 @@
 		"querystringify": "1.0.0",
 		"re-resizable": "4.0.3",
 		"react": "16.2.0",
-		"react-autosize-textarea": "2.0.0",
+		"react-autosize-textarea": "3.0.2",
```

Comment:
Was this meant to be added here?

Reply:
Originally I thought the bug was specific to this PR,I can remove the commit from this PR.